benchmark/src/main/scala/spray/json/GithubIssuesJsonAST.scala

```diff
@@ -0,0 +1,54 @@
+package spray.json
+
+import org.openjdk.jmh.annotations.Benchmark
+import org.openjdk.jmh.annotations.Setup
+
+import scala.io.Source
+
+import play.api.libs.json.Json
+
+class GithubIssuesJsonAST extends Common {
+  var jsonString: String = _
+  var jsonBytes: Array[Byte] = _
+  var sprayJsonAST: JsValue = _
+  var playJsonAST: play.api.libs.json.JsValue = _
+  var upickleAST: ujson.Js.Value = _
```

Comment:
uJson is vulnerable for hash collision attacks. It was reported in https://github.com/lihaoyi/upickle/issues/273 but @lihaoyi don't want to fix it and even removed the issue with all input from the Scala community about possible solutions how to do it without lost in efficiency.

Btw, code to reproduce the problem is available in [this PR](https://github.com/plokhotnyuk/jsoniter-scala/pull/325)

Reply:
This is not the place to discuss that issue.

I had this benchmark branch running for a while so I included upickle back then. In fact, there's not so much value at all in including other libraries, it's more of a base line to see where we stand and where we might be able to go.